openscad_docsgen/blocks.py

```diff
@@ -390,6 +394,12 @@ def get_markdown(self, controller):
         out.append("## {}".format(mkdn_esc(str(self))))
         out.extend(self.get_markdown_body(controller))
         out.append("")
+        out += self.get_childrens_mardown(controller)
+
+        return out
+
+    def get_childrens_mardown(self, controller):
```

Comment:
spelling typo.  Should have a  in get_childrens_mardown()

Reply:
Should be fixed. I have also added doc strings to the methods I created or modified. I have then rebased of main.